packages/client/src/components/ViewDropDown/ViewDropdown.component.js

```diff
@@ -12,11 +13,19 @@ const ViewDropdown = ({ options, label }) => {
         </option>
       );
     });
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+    onSelect(event.target.value);
```

Comment:
Why are we setting a value and calling a callback? Did you consider making value a prop?

Reply:
I am getting value onSelect function from dropdown and second one updated state and store value.